endpoints/openrtb2/auction.go

```diff
@@ -1946,3 +1912,18 @@ func checkIfAppRequest(request []byte) (bool, error) {
 	}
 	return false, nil
 }
+
+func validateStoredBidRespAndImpExtBidders(bidderExts map[string]json.RawMessage, storedBidResp stored_responses.ImpBidderStoredResp, impId string) []error {
+	if bidResponses, ok := storedBidResp[impId]; ok {
+		storedBidRespValidationError := []error{fmt.Errorf("request validation failed. Stored bid responses are specified for imp %s. Bidders specified in imp.ext should match with bidders specified in imp.ext.prebid.storedbidresponse", impId)}
+		if len(bidResponses) != len(bidderExts)-1 { // do not count "prebid" part of imp.ext. It is always present in imp.ext if stored bid responses are specified
```

Comment:
Should this be a `>` instead of `!=`?
I think an imp bidder can exist in the request but not have a stored response. Every bidder that has a stored response though should be an imp bidder.

Reply:
Right, should be `if len(bidResponses) < len(bidderExts)-1 `